README.md

```diff
@@ -132,4 +141,4 @@ inNative is compiled as either a dynamic or static library, and can be integrate
     (*update_entity)(0);
     
     // Free assembly once finished
-    (*exports.FreeAssembly)(assembly);
\ No newline at end of file
+    (*exports.FreeAssembly)(assembly);
```

Comment:
The heck is going on with this change? Can we slightly adjust the commit so it doesn't show up?

Reply:
Are you referring to the last line? I believe vim automatically inserted a newline when I edited the file since it treats newlines as line terminators rather than line separators.